app/controllers/static_pages_controller.rb

```diff
@@ -1,6 +1,5 @@
 class StaticPagesController < ApplicationController
   def home
-    @navbar = false
     @is_home_page = true
```

Comment:
They are flash messages intended to be used for announcements for all users on the site. With the homepage not really being accessible to signed in users now we will want to show them on all pages if we decide to keep that feature.